tests/regf.py

```diff
@@ -69,26 +70,31 @@ def data(self):
 class REGFWinRegistryFileTest(test_lib.BaseTestCase):
   """Tests for the REGF Windows Registry file."""
 
-  @test_lib.skipUnlessHasTestFile(['NTUSER.DAT'])
   def testOpenClose(self):
     """Tests the Open and Close functions."""
-    path = self._GetTestFilePath(['NTUSER.DAT'])
+    test_path = self._GetTestFilePath(['NTUSER.DAT'])
```

Comment:
Please move the skip functionality to a method in the base test case, rather than repeating this logic.